core/services/keeper/orm.go

```diff
@@ -154,27 +156,20 @@ func (korm ORM) CreateEthTransactionForUpkeep(ctx context.Context, upkeep Upkeep
 			WHERE eth_txes.from_address = $1
 				AND eth_txes.state = 'unconfirmed'
 				AND eth_tx_attempts.state = 'insufficient_eth'
-		);`,
+		) RETURNING id;`,
 		from,
 		upkeep.Registry.ContractAddress.Address(),
 		payload,
 		value,
 		upkeep.ExecuteGas+gasBuffer,
-	)
-
+	).Scan(&etx.ID)
```

Comment:
In the case of insufficient eth this will scan a null (0) value. We probably want to check for that and return some sort of "insufficient eth" error.